crates/ide_db/src/imports/insert_use/tests.rs

```diff
@@ -86,7 +86,7 @@ use external_crate2::bar::A;",
 
 #[test]
 fn insert_not_group_empty() {
-    cov_mark::check!(insert_no_grouping_last2);
+    cov_mark::check!(insert_empty_file);
```

Comment:
Reusing the same label is a bad idea, these tests can run in parallel so this could succeed without being hit by this test

Reply:
@Veykril Is there a preferred way how the tests should be structured, if two tests execute the same code path and thus should hit the same points? Should I merge the two test functions `insert_not_group_empty` and `insert_empty_file` into one bigger one?

Reply:
Ye since they now hit the same path we can put those into the same test function